components/quickbooks/sources/custom-webhook-events/custom-webhook-events.js

```diff
@@ -0,0 +1,48 @@
+const quickbooks = require("../../quickbooks.app");
+const common = require("../common");
+
+module.exports = {
+  ...common,
+  key: "quickbooks-custom-webhook-events",
+  name: "Custom Set of Webhook Entities (Created, Updated, Merged, Deleted, Voided or Emailed)",
+  description: "Emit events for more than one type of entity (e.g. \"Emailed Invoices and Purchase Orders\" or \"New and Modified Customers and Vendors\"). Visit the documentation page to learn how to configure webhooks for your QuickBooks company: https://developer.intuit.com/app/developer/qbo/docs/develop/webhooks",
```

Comment:
**-Minor-**
The [linter](https://pipedream.com/docs/components/guidelines/#local-checks) shows a warning for the `description` on this line because it doesn't start with ["Emit new"](https://pipedream.com/docs/components/guidelines/#source-description). To fix this, we could change this line to:
```js
  description: "Emit new event for selected operations on entities (e.g. \"Emailed Invoices and Purchase Orders\" or \"New and Modified Customers and Vendors\"). Visit the documentation page to learn how to configure webhooks for your QuickBooks company: https://developer.intuit.com/app/developer/qbo/docs/develop/webhooks",
```